### PR TITLE
Change RoomData to booleans

### PR DESCRIPTION
**Pull request by matotubol**

### CODE REVIEW

File: Resources/SQLs/Updates/9_ChangeRoomDataToBooleans.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE `rooms` CHANGE `allow_pets` `allow_pets` BOOLEAN NOT NULL DEFAULT FALSE, CHANGE `allow_pets_eat` `allow_pets_eat` BOOLEAN NOT NULL DEFAULT FALSE, CHANGE `room_blocking_disabled` `room_blocking_disabled` BOOLEAN NOT NULL DEFAULT FALSE, CHANGE `allow_hidewall` `allow_hidewall` BOOLEAN NOT NULL DEFAULT FALSE;
```
Comment (80O) [inline Resources/SQLs/Updates/9_ChangeRoomDataToBooleans.sql:1]: Append this to the end of the original SQL file.

File: HabboHotel/Rooms/RoomData.cs [hunk 2]
```diff
@@ -129,13 +129,13 @@ public RoomData(RoomData data)
...
+    public bool AllowPets { get; set; }
     // TODO @80O: Convert to bool
-    public int AllowPetsEating { get; set; }
+    public bool AllowPetsEating { get; set; }
     // TODO @80O: Convert to bool
-    public int RoomBlockingEnabled { get; set; }
+    public bool RoomBlockingEnabled { get; set; }
     // TODO @80O: Convert to bool
-    public int Hidewall { get; set; }
+    public bool Hidewall { get; set; }
```
Comment (80O) [inline HabboHotel/Rooms/RoomData.cs:138]: You can remove the TODO's aswell

File: Communication/Packets/Incoming/Rooms/Settings/SaveRoomSettingsEvent.cs [hunk 3]
```diff
@@ -50,10 +50,10 @@ public Task Parse(GameClient session, IIncomingPacket packet)
             formattedTags.Append(tag);
         }
         var tradeSettings = packet.ReadInt(); //2 = All can trade, 1 = owner only, 0 = no trading.
-        var allowPets = packet.ReadBool() ? 1 : 0;
-        var allowPetsEat = packet.ReadBool() ? 1 : 0;
-        var roomBlockingEnabled = packet.ReadBool() ? 1 : 0;
-        var hidewall = packet.ReadBool() ? 1 : 0;
+        var allowPets = packet.ReadBool() ? true : false;
```
Comment (80O) [inline Communication/Packets/Incoming/Rooms/Settings/SaveRoomSettingsEvent.cs:53]: Ternary operator not needed, ReadBoolal returns a bool ;)

**Comment by matotubol** [comment]
Done.

**Pull request merged**